hallux/targets/github_proposal.py

```diff
@@ -120,9 +121,10 @@ def commit_diff(self) -> bool:
                 side="RIGHT",
                 path=str(compacted.filename),
                 line=compacted.end_line,
-                start_line=compacted.start_line,
+                start_line=compacted.start_line if compacted.start_line > compacted.end_line else GithubObject.NotSet,
             )
-        except GithubException:
+        except BaseException as ex:
```

Comment:
Catch a more specific exception or reraise the exception
```suggestion
        except Exception as ex: 

```